Classes/GlobalStateExplorers/DatabaseBrowser/FLEXTableContentViewController.m

```diff
@@ -0,0 +1,183 @@
+  }
+  return self;
+}
+
+- (void)viewWillAppear:(BOOL)animated
+{
+  [super viewWillAppear:animated];
+  [self.multiColumView reloadData];
+  
+}
+
+#pragma mark -
+#pragma mark MultiColumnTableView DataSource
+
+- (NSInteger)numberOfColumnsInTableView:(FLEXMultiColumnTableView *)tableView
+{
+  return self.columnsArray.count;
+}
+- (NSInteger)numberOfRowsInTableView:(FLEXMultiColumnTableView *)tableView
+{
+  return self.contensArray.count;
+}
+
+
+- (NSString *)columnNameInColumn:(NSInteger)column
+{
+    return self.columnsArray[column];
+}
+
+
+- (NSString *)rowNameInRow:(NSInteger)row
+{
+  return [NSString stringWithFormat:@"%ld",(long)row];
+}
+
+- (NSString *)contentAtColumn:(NSInteger)column row:(NSInteger)row
+{
+  if (self.contensArray.count > row) {
+    NSDictionary *dic = self.contensArray[row];
+    if (self.contensArray.count > column) {
+      return [NSString stringWithFormat:@"%@",[dic objectForKey:self.columnsArray[column]]];
+    }
+  }
+  return @"";
+}
+
+- (NSArray *)contentAtRow:(NSInteger)row
+{
+  NSMutableArray *result = [NSMutableArray array];
+  if (self.contensArray.count > row) {
+    NSDictionary *dic = self.contensArray[row];
+    for (int i = 0; i < self.columnsArray.count; i ++) {
+      [result addObject:dic[self.columnsArray[i]]];
+    }
+    return  result;
+  }
+  return nil;
+}
+
+- (CGFloat)multiColumnTableView:(FLEXMultiColumnTableView *)tableView
+      heightForContentCellInRow:(NSInteger)row
+{
+  return 40;
+}
+
+- (CGFloat)multiColumnTableView:(FLEXMultiColumnTableView *)tableView
+    widthForContentCellInColumn:(NSInteger)column
+{
+  return 120;
+}
+
+- (CGFloat)heightForTopHeaderInTableView:(FLEXMultiColumnTableView *)tableView
+{
+  return 40;
+}
+
+- (CGFloat)WidthForLeftHeaderInTableView:(FLEXMultiColumnTableView *)tableView
+{
+  NSString *str = [NSString stringWithFormat:@"%lu",(unsigned long)self.contensArray.count];
+  NSDictionary *attrs = @{@"NSFontAttributeName":[UIFont systemFontOfSize:17.0]};
+  CGSize size =   [str boundingRectWithSize:CGSizeMake(CGFLOAT_MAX, 14)
+                                    options:NSStringDrawingUsesLineFragmentOrigin
+                                 attributes:attrs context:nil].size;
+  return size.width + 20;
+}
+
+#pragma mark -
+#pragma mark MultiColumnTableView Delegate
+
+- (void)multiColumnTableView:(FLEXMultiColumnTableView *)tableView labelDidTapWithText:(NSString *)text
+{
+  FLEXWebViewController * detailViewController = [[FLEXWebViewController alloc] initWithText:text];
+  [self.navigationController pushViewController:detailViewController animated:YES];
+}
+
+- (void)multiColumnTableView:(FLEXMultiColumnTableView *)tableView headerTapWithText:(NSString *)text sortType:(FLEXTableColumnHeaderSortType)type
+{
+  
+  NSArray *sortContentData = [self.contensArray sortedArrayUsingComparator:^NSComparisonResult(id obj1, id obj2) {
+    
+    if ([obj1 objectForKey:text] == [NSNull null]) {
+      return 0;
```

Comment:
Can we use the `NSComparisonResult` enum values here? Also, 0 corresponds to `NSOrderedSame` - I'm not sure that's what we want here.